src/main/java/com/odysseusinc/arachne/portal/security/AuthenticationTokenFilter.java

```diff
@@ -89,8 +90,6 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                         if (!HttpMethod.OPTIONS.matches(method)) {
                             if (log.isDebugEnabled()) {
```

Comment:
this if is not required if we want to remove else part. But it can make sense to change else to info or warn